code/__HELPERS/mobs.dm

```diff
@@ -775,6 +775,18 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 		return TRUE
 	return FALSE
 
+//// Special handler for cyborg naming to ensure no cyborgs ever take a name that has already been used. Prevents "respawning" when the same player takes two posibrains.
+/mob/proc/apply_cyborg_name(preference_type, client/C)
+
+	//This name has already been taken, refuse to pass it on and return false
+	if(preference_type in GLOB.cyborg_name_list)
```

Comment:
This seems like it will just add and search for the typepath `/datum/preference/name/cyborg` to the name list, causing all future joins after the first player to be randomized since it isn't actually accounting for the name itself.

Reply:
Valid concern, might be a day or a few before I can address

Reply:
This is now resolved.

Reply:
Is it? The code looks exactly the same as it was. Did you forget to push the changes?

Reply:
I have no idea what you're talking about!
I would never forget something so silly as forgetting to push my changes